Documentation/Troubleshooting.md

```diff
@@ -199,4 +199,17 @@ Fire attach
 System.Diagnostics.Debugger.Launch();
 ```
 
+If you want debug in-process collector you need to set VSTEST_HOST_DEBUG(https://github.com/microsoft/vstest/issues/2158) environment variable
+```
+set VSTEST_HOST_DEBUG=1
+```
+Test host will wait for debugger
+```
+Starting test execution, please wait...
+Logging Vstest Diagnostics in file: C:\git\coverletissue\collectorlog\XUnitTestProject1\log.txt
```

Comment:
Nit `VSTest`

Reply:
I cannot change that, it's a message emitted by vs test runtime